diagnostic.js

```diff
@@ -32,25 +32,93 @@ const Movement = require('./models/movement.js');
 
 /// ADD YOUR CODE BELOW
 
-const create = (name, description, startYear, endYear) => {};
+const done = function() {
+  db.close();
+};
+
+const create = (name, description, startYear, endYear) => {
+  Movement.create({
+    name : name,
+    description : description,
+    startYear : startYear,
+    endYear: endYear,
+  })
+  .then(()=>console.log(movement.toJSON))
```

Comment:
".then(()=>console.log(movement.toJSON))" should be ".then(movement=>console.log(movement.toJSON))" otherwise movement is undefined.